frontend/public/locales/zh/public.json

```diff
@@ -910,7 +901,7 @@
   "Enter the name of the {{label}} to delete": "输入要删除的 {{label}} 名称",
   "Delete PersistentVolumeClaim": "删除持久性卷声明",
   "Are you sure you want to delete <2>{{pvcName}}</2> PersistentVolumeClaim?": "确定要删除 <2>{{pvcName}}</2> 持久性卷声明吗？",
-  "OK": "确定",
+  "OK": "OK",
```

Comment:
Not sure why this was updated to EN. Will check this with the localization team.